Extension/gulpfile.js

```diff
@@ -6,15 +6,45 @@
 'use strict';
 
 const gulp = require('gulp');
+const env = require('gulp-env')
 const tslint = require('gulp-tslint');
 const mocha = require('gulp-mocha');
 
-gulp.task('unitTest', () => {
+gulp.task('allTests', () => {
+    gulp.start('unitTests');
+    gulp.start('integrationTests');
+});
+
+gulp.task('unitTests', () => {
     gulp.src('./out/test/unitTests', {read: false}).pipe(
         mocha({
             ui: "tdd"
         })
+    ).once('error', err => {
+        process.exit(1);
+    })
+    .once('end', () => {
+        process.exit();
+    })
+});
+
+gulp.task('integrationTests', () => {
```

Comment:
since we have a sample project in the repo can we reuse that instead of creating a new one?

Reply:
No. :)